packages/@atjson/document/test/atjson-test.ts

```diff
@@ -112,42 +111,31 @@ describe('new Document', () => {
     ];
 
     test('non-global regex returns first match', () => {
-      let matches = document.match(/and/);
-      expect(matches).toEqual(MATCHES_AND.slice(0,1));
+      expect(document.match(/and/)).toEqual(MATCHES_AND.slice(0,1));
     });
 
     test('global regex returns all matches', () => {
-      let matches = document.match(/and/g);
-      expect(matches).toEqual(MATCHES_AND);
+      expect(document.match(/and/g)).toEqual(MATCHES_AND);
     });
 
     test('match groups are ok but don\'t affect matches returned', () => {
-      let matches = document.match(/(a)(nd)+/g);
-      expect(matches).toEqual(MATCHES_AND);
+      expect(document.match(/(a)(nd)+/g)).toEqual(MATCHES_AND);
     });
 
     test('regex can contain unicode characters', () => {
-      let matches = document.match(/[\u000B\u220E]/g);
-      expect(matches).toEqual([
+      expect(document.match(/[\u000B\u220E]/g)).toEqual([
         { start: 594, end: 595 },
         { start: 595, end: 596 },
         { start: 1270, end: 1271 }
       ]);
     });
 
     test('match finds within ranges', () => {
-      let matches = document.match(/and/g, undefined, 500);
-      expect(matches).toEqual(MATCHES_AND.slice(0, 3));
-
-      matches = document.match(/and/g, 500);
-      expect(matches).toEqual(MATCHES_AND.slice(3));
-
-      matches = document.match(/and/g, 500, 505);
-      expect(matches).toEqual([]);
-
-      matches = document.match(/and/g, 500, 800);
-      expect(matches).toEqual(MATCHES_AND.slice(3, 7));
-    })
+      expect(document.match(/and/g, undefined, 500)).toEqual(MATCHES_AND.slice(0, 3));
```

Comment:
👍 Hm I should probably just be passing 0 here instead of `undefined`. Would prefer to not pass in some sort of 'range' object that we'd have to check for null/undefined

Reply:
`0` is probably best!